--- qwt-svn/src/qwt_spline.h.pyqwt	2007-01-03 16:18:36.000000000 +0100
+++ qwt-svn/src/qwt_spline.h	2007-01-03 16:21:20.000000000 +0100
@@ -95,10 +95,12 @@
 
 #if QT_VERSION < 0x040000
     bool setPoints(const QwtArray<QwtDoublePoint>& points);
-    QwtArray<QwtDoublePoint> points() const;
+    // points() is not implemented.
+    // QwtArray<QwtDoublePoint> points() const;
 #else
     bool setPoints(const QPolygonF& points);
-    QPolygonF points() const;
+    // points() is not implemented.
+    // QPolygonF points() const;
 #endif
 
     void reset();
