--- qwt-cvs/include/qwt_spline.h.pyqwt	2006-09-10 12:30:17.000000000 +0200
+++ qwt-cvs/include/qwt_spline.h	2006-09-10 12:32:03.000000000 +0200
@@ -92,10 +92,12 @@
 
 #if QT_VERSION < 0x040000
     bool setPoints(const QwtArray<QwtDoublePoint>& points);
-    QwtArray<QwtDoublePoint> points() const;
+    // points() is not implemented.
+    // QwtArray<QwtDoublePoint> points() const;
 #else
     bool setPoints(const QPolygonF& points);
-    QPolygonF points() const;
+    // points() is not implemented.
+    // QPolygonF points() const;
 #endif
 
     void reset();
