--- qwt-5.2/qwtconfig.pri.pyqwt Sat Apr 18 05:45:43 2009
+++ qwt-5.2/qwtconfig.pri Sat Jun  6 06:15:51 2009
@@ -75,7 +75,7 @@
 # it will be a static library.
 ######################################################################
 
-CONFIG           += QwtDll
+# CONFIG           += QwtDll
 
 ######################################################################
 # QwtPlot enables all classes, that are needed to use the QwtPlot 
@@ -97,7 +97,7 @@
 # QwtSVGItem.
 ######################################################################
 
-#CONFIG     += QwtSVGItem
+CONFIG     += QwtSVGItem
 
 ######################################################################
 # If you have a commercial license you can use the MathML renderer
--- qwt-5.2/src/qwt_global.h.pyqwt Sat Apr 18 05:45:42 2009
+++ qwt-5.2/src/qwt_global.h Sat Jun  6 06:16:03 2009
@@ -19,8 +19,8 @@
 
 // QWT_VERSION is (major << 16) + (minor << 8) + patch.
 
-#define QWT_VERSION       0x050200
-#define QWT_VERSION_STR   "5.2.0"
+#define QWT_VERSION       0x050201
+#define QWT_VERSION_STR   "5.2.1"
 
 #if defined(Q_WS_WIN)
 
