--- qwt-cvs/include/qwt_color_map.h.pyqwt	2006-03-25 14:21:11.000000000 +0100
+++ qwt-cvs/include/qwt_color_map.h	2006-03-25 14:22:27.000000000 +0100
@@ -19,7 +19,7 @@
 #endif
 #include "qwt_double_interval.h"
 
-class QwtColorMap
+class QWT_EXPORT QwtColorMap
 {
 public:
     enum Format
@@ -54,7 +54,7 @@
 };
 
 
-class QwtContinousColorMap: public QwtColorMap
+class QWT_EXPORT QwtContinousColorMap: public QwtColorMap
 {
 public:
     QwtContinousColorMap(QwtColorMap::Format = QwtColorMap::RGB);
